TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/KafkaPrinter.java

```diff
@@ -34,9 +40,42 @@
  * 3. Clearing records for a topic with the same group id
  */
 public class KafkaPrinter {
+
     private static final Logger log = LoggerFactory.getLogger(KafkaPrinter.class);
     public static final Duration CONSUMER_POLL_TIMEOUT = Duration.ofSeconds(1);
 
+    static class Partition {
```

Comment:
There's already a tuple class for topics and partitions in org.apache.kafka.common (TopicPartition)